src/web/make-report.rkt

```diff
@@ -61,9 +61,18 @@
     (for/sum ([t tests]) (or (table-row-start t) 0)))
   (define total-result
     (for/sum ([t tests]) (or (table-row-result t) 0)))
-  (define total-bits
-    (for/sum ([t tests])
-      (representation-total-bits (get-representation (table-row-precision t)))))
+  (match-define
+    (list
+     (list _ maximum-accuracy)
+     (list _ initial-accuracy)
+     frontier)
+    (merged-cost-accuracy tests))
+  (define speedup-at-initial-accuracy
+    (match
+        (findf
+         (match-lambda [(list _ accuracy) (> accuracy initial-accuracy)])
+         frontier)
+    [(list cost _) (/ 1 cost)]))
```

Comment:
Ok, all of these `match`es would be more readable as either `first`/`second` or, if that feels too obscure, by defining `point-cost` and `point-accuracy` helpers. This could be:

```
(/ 1 (point-cost (findf (lambda (pt) (> (point-accuracy pt) initial-accuracy)) frontier)))
```